src/api/spec/models/package_spec.rb

```diff
@@ -834,4 +834,11 @@
       it { expect { subject }.to raise_error(Package::CycleError) }
     end
   end
+
+  describe '.multibuild_flavor?' do
+    let(:multibuild_package) { create(:multibuild_package, name: 'multibuild_package', project: home_project, flavors: ['flavor_a', 'flavor_b']) }
+
+    it { expect(multibuild_package).to be_multibuild_flavor('flavor_a') }
```

Comment:
```suggestion
    let(:multibuild_package) { create(:multibuild_package, name: 'multibuild_package', project: home_project, flavors: ['flavor_a', 'flavor_b'], package: ['flavor_c']) }

    it { expect(multibuild_package).to be_multibuild_flavor('flavor_a') }
    it { expect(multibuild_package).to be_multibuild_flavor('flavor_c') }
```

The code is introducing parsing of `package` elements. Shouldn't we also add this to the tests, as suggested? In that case we should also modify the multibuild_package factory.